package.json

```diff
@@ -90,6 +90,7 @@
     "js-beautify": "^1.6.4",
     "jsdom": "^9.8.3",
     "jshint": "^2.9.4",
+    "just-compare": "^1.2.1",
```

Comment:
since this project already includes libraries like `lodash`, does it make sense to add another dependency?

Reply:
remove that~Thanks for reminding!